src/operator/src/statement/copy_table_from.rs

```diff
@@ -379,11 +379,14 @@ impl StatementExecutor {
 
         let mut rows_inserted = 0;
         let mut insert_cost = 0;
-        let max_insert_rows = req
-            .with
-            .get(MAX_INSERT_ROWS)
-            .and_then(|val| val.parse::<usize>().ok())
-            .unwrap_or(DEFAULT_MAX_INSERT_ROWS);
+        let max_insert_rows = if let Some(num) = req.limit {
+            num as usize
+        } else {
+            req.with
+                .get(MAX_INSERT_ROWS)
+                .and_then(|val| val.parse::<usize>().ok())
+                .unwrap_or(DEFAULT_MAX_INSERT_ROWS)
+        };
```

Comment:
Is there a way to disable the limit?

Reply:
Maybe we can add `LIMIT 0` to disable it.

Reply:
@waynexia What do you think?

Reply:
Maybe we should not limit if no `LIMIT` is specified

>Change the limit behavior to
>- do not limit by default
>- make the limit rows configurable

Reply:
> Maybe we should not limit if no LIMIT is specified

Makes sense.